packages/next/build/index.ts

```diff
@@ -55,20 +55,23 @@ export default async function build(dir: string, conf = null): Promise<void> {
   const pagePaths = await collectPages(pagesDir, config.pageExtensions)
   const pages = createPagesMapping(pagePaths, config.pageExtensions)
   const entrypoints = createEntrypoints(pages, config.target, buildId, config)
+  const appPath = pages['/_app'] || 'next/dist/pages/_app'
```

Comment:
`createPagesMapping` already defaults the value to always be there: https://github.com/zeit/next.js/blob/canary/packages/next/build/entries.ts#L19